llvm/utils/lit/tests/substitutions.py

```diff
@@ -0,0 +1,5 @@
+# Basic test for substitutions.
+#
+# RUN: echo %basename_s  | FileCheck %s --check-prefix=BASENAME
+#
+# BASENAME: substitutions.py
```

Comment:
For a file w/ a single check, I think `CHECK` is probably a better choice over a specific prefix. If you plan to follow up w/ more tests, then using the prefix makes sense though, so I'll leave that up to you to decide on the better approach.

Reply:
Thanks, changed it back to just CHECK. I did look at adding more checks, but some things like full path, it wasn't clear how to check that. So, restricting the test to just the new substitutions being added. if/when its extended for others, the check prefix can be updated.